iothub/device/src/DeviceClientOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// Options that allow configuration of the device client instance during initialization.
+    /// </summary>
+    public class DeviceClientOptions
+    {
+        /// <summary>
+        /// The Plug and Play Model Id associated with the device identity.
+        /// </summary>
+        public string PlugAndPlayModelId { get; set; } = default;
```

Comment:
`= default;` is not required

Reply:
Good point, there is no need for a default value to be associated here, will remove.